script.js

```diff
@@ -0,0 +1,72 @@
+/* eslint-disable no-unused-vars */
+let books = [];
+
+// retrieve saved books
+function retrieve() {
+  const data = JSON.parse(localStorage.getItem('aboutBook'));
+  if (data) {
+    books = JSON.parse(localStorage.getItem('aboutBook'));
+  }
+}
+retrieve();
+
+const main = document.querySelector('.books');
+
+function addBook(enteredTitle, enteredAuthor) {
+  books.push({ title: enteredTitle, author: enteredAuthor });
+}
```

Comment:
- [ ] Please make sure that when a user clicks the "Add" button:
- A new book is added to the collection.
- The new book is displayed on the page.